docs/UserGuide.md

```diff
@@ -16,13 +16,13 @@ This is a mockup of our UI.
 - [Features](#features)
     - [Viewing help](#viewing-help)
     - [Contact](#adding-a-contact)
```

Comment:
Can add hyperlinks to each command under the Contact category, same for Entry category.